pkg/cmd/pipelinerun/describe_test.go

```diff
@@ -669,52 +805,105 @@ func TestPipelineRunsDescribe_custom_output(t *testing.T) {
 	prun := []*v1alpha1.PipelineRun{
 		tb.PipelineRun(pipelinerunname, "ns"),
 	}
-	clock := clockwork.NewFakeClock()
-	cs, _ := test.SeedTestData(t, pipelinetest.Data{
-		PipelineRuns: prun,
-		Namespaces: []*corev1.Namespace{
-			{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: "ns",
-				},
+	namespaces := []*corev1.Namespace{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "ns",
 			},
 		},
-	})
+	}
 
-	p := &test.Params{Tekton: cs.Pipeline, Clock: clock, Kube: cs.Kube}
-	pipelinerun := Command(p)
+	version := "v1alpha1"
+	tdc := testDynamic.Options{}
+	dynamic, err := tdc.Client(
+		cb.UnstructuredPR(prun[0], version),
+	)
+	if err != nil {
+		t.Errorf("unable to create dynamic client: %v", err)
+	}
+	cs, _ := test.SeedTestData(t, pipelinetest.Data{Namespaces: namespaces, PipelineRuns: prun})
+	cs.Pipeline.Resources = cb.APIResourceList(version, []string{"pipelinerun", "taskrun"})
+	p := &test.Params{Tekton: cs.Pipeline, Kube: cs.Kube, Dynamic: dynamic}
 
+	pipelinerun := Command(p)
 	got, err := test.ExecuteCommand(pipelinerun, "desc", "-o", "name", "-n", "ns", pipelinerunname)
 	if err != nil {
 		t.Errorf("Unexpected error: %v", err)
 	}
+	got = strings.TrimSpace(got)
+	if got != expected {
+		t.Errorf("Result should be '%s' != '%s'", got, expected)
+	}
+}
+
+func TestPipelineRunDescribeV1beta1_custom_output(t *testing.T) {
```

Comment:
Can you add another unit test for a `v1beta1` pipelinerun in a non `-o` case? Perhaps a pipelinerun with params, resources, and taskruns that is compared against a golden file?

Reply:
Thanks, added